src/js_native_api_v8.h

```diff
@@ -8,14 +8,15 @@ inline napi_status napi_clear_last_error(node_api_nogc_env env);
 
 namespace v8impl {
 
+// Base class to track references and finalizers in a double linked list.
```

Comment:
```suggestion
// Base class to track references and finalizers in a doubly linked list.
```

Reply:
Thanks! I have added this change to the latest commit.